nuplan_garage/planning/script/config/common/model/pdm_offset_model.yaml

```diff
@@ -0,0 +1,49 @@
+_target_: nuplan_garage.planning.training.modeling.models.pdm_offset_model.PDMOffsetModel
+_convert_: 'all'
+
+trajectory_sampling:
+  _target_: nuplan.planning.simulation.trajectory.trajectory_sampling.TrajectorySampling
+  _convert_: 'all'
+  num_poses: 16      # target future poses
+  interval_length: 0.5 
+
+history_sampling:
+  _target_: nuplan.planning.simulation.trajectory.trajectory_sampling.TrajectorySampling
+  _convert_: 'all'
+  num_poses: 10      # target future poses
+  interval_length: 0.2  
+
+planner:
+  _target_: nuplan_garage.planning.simulation.planner.pdm_planner.pdm_closed_planner.PDMClosedPlanner
+  _convert_: 'all'
+  
+  # parameters for output trajectory
+  trajectory_sampling:
+    _target_: nuplan.planning.simulation.trajectory.trajectory_sampling.TrajectorySampling
+    _convert_: 'all'
+    num_poses: 80 # target future poses
+    interval_length: 0.1 # interval of samples [s]
+  
+  # parameters for proposals
+  proposal_sampling:
+    _target_: nuplan.planning.simulation.trajectory.trajectory_sampling.TrajectorySampling
+    _convert_: 'all'
+    num_poses: 40 # target future poses
```

Comment:
Let's add a comment that we decide to use 40 instead of 80 for runtime reasons (and since the first seconds are more relevant and meaningful based on the cv forecasting)